### PR TITLE
Query account by public key

### PR DESCRIPTION
**Pull request by tifrel**

### CODE REVIEW

**Review by sainthiago — APPROVED** [review]

File: packages/data/src/api/accountsByPublicKey/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### accountsByPublicKey(publicKey: string): Promise<ParsedDataReturn<string[]>>
+
+Example:
+
+
+
+{% code title="accountsByPublicKey.ts" overflow="wrap" lineNumbers="true" %}
+
+```typescript
```
Comment (bh2smith) [inline packages/data/src/api/accountsByPublicKey/README.md:23]: FYI: I believe you can also go with just ts here:

```suggestion
```ts
```

File: packages/data/src/api/accountsByPublicKey/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import { accountsByPublicKey } from  '@mintbase-js/data'
+
+const { data, error } = await accountsByPublicKey('ed25519:12345...');
+
+if (error) {console.log('error', error)}
```
Comment (bh2smith) [inline packages/data/src/api/accountsByPublicKey/README.md:29]: ubernit: 
```suggestion
if (error) {console.error(error)}
```

**Review by bh2smith — APPROVED** [review]
Looks great!

**Review by SurgeCode — APPROVED** [review]

File: packages/data/src/api/accountsByPublicKey/README.md (hunk 1 above)
Comment (tifrel) [reply packages/data/src/api/accountsByPublicKey/README.md:23]: Yup, but this is consistent with the rest of documentation (I just copied the template from another README)

File: packages/data/src/api/accountsByPublicKey/README.md (hunk 2 above)
Comment (tifrel) [reply packages/data/src/api/accountsByPublicKey/README.md:29]: See above, we'll keep it consistent with other docs

**Pull request merged**